typescript/test/parser.bench.ts

```diff
@@ -0,0 +1,19 @@
+import { bench, describe } from "vitest";
+import { CooklangParser } from "../src/parser";
+
+const recipeString = "Make your first @recipe!";
+describe("parser", () => {
+  bench("instance", () => {
+    const recipe = new CooklangParser(recipeString);
+  });
+
+  // init the parser outside of the bench which
+  // technically saves a few cycles on the actual
+  // bench result vs the instance bench, but this is
+  // effectively the use case where you init once
+  // and reuse that parser over and over
```

Comment:
Good to point out that this is a major optimization! I'm thinking the "right way" should be the way we guide users towards through the library design. That's why I used the `static` parser in #60.

To say it another way: I don't see the benefit of "instance" parsing. What do you think?